static/app/components/segmentedControl.tsx

```diff
@@ -129,7 +129,7 @@ function Segment({
       {isSelected && (
         <SegmentSelectionIndicator
           layoutId={layoutGroupId}
-          transition={{type: 'tween', ease: 'circOut', duration: 0.2}}
+          transition={{type: 'tween', ease: 'easeOut', duration: 0.2}}
```

Comment:
Different easing curve with less of a sudden bounce, based on feedback from @JonasBa.